packages/cluster-ui/src/tableStatistics/tableStatistics.tsx

```diff
@@ -16,12 +22,30 @@ interface TableStatistics {
   activeFilters: number;
   search?: string;
   onClearFilters?: () => void;
+  resetSQLStats: () => void;
 }
 
+const toolTipText = `Statement history is cleared once an hour by default, which can be configured with the cluster setting
+ diagnostics.reporting.interval. Clicking ‘Clear SQL stats’ will reset SQL stats on the statements and transactions pages.`;
+
 const renderLastCleared = (lastReset: string | Date) => {
   return `Last cleared ${moment.utc(lastReset).format(DATE_FORMAT)}`;
 };
 
+const renderResetSQLStats = (resetSQLStats: () => void) => {
```

Comment:
for both render reset stats and tooltip there is no need to pass parameters (since they're both static) and they look very simple, so I don't think is worth creating a render function for each one (the last cleared render requires some "calculation", so it's better to leave it separated), but I would suggest add those two renders directly

Reply:
Done